### PR TITLE
#35 create http service

### LINKED ISSUE #35: Create HTTP service
The client should have an HTTP service that will `http.post` a user command to the server.

### PR DESCRIPTION
**Pull request by ZimGil**
This PR uses an `HttpClient` service to `http.post` the commands to the server.
for now it's only printing the response

resolves #35

### CODE REVIEW

File: src/app/models/angular-command-request.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { AngularCliCommand } from './angular-cli-command.interface';
```
Comment (ortichon) [inline src/app/models/angular-command-request.ts]: ?

File: src/app/services/command/command.service.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { CommandRequest } from '../../models/angular-command-request';
+
+@Injectable({
+  providedIn: 'root'
```
Comment (ortichon) [inline src/app/services/command/command.service.ts]: please check if we need this line.
I don't use it in my projects. 
I don't understand what it's for.

Comment (ZimGil) [reply src/app/services/command/command.service.ts]: [https://angular.io/guide/dependency-injection#create-an-injectable-heroservice](url)

>   // we declare that this service should be created
>   // by the root application injector.
>

Comment (ZimGil) [reply src/app/services/command/command.service.ts]: Is `'root'` the default value?
[https://angular.io/api/core/Injectable](url)

**Pull request merged**